packages/seatmaps-client/package.json

```diff
@@ -18,9 +18,15 @@
   "repository": "github:ticketevolution/seatmaps-client",
   "author": "Ticket Evolution, Inc.",
   "license": "MIT",
-  "main": "./cjs/index.js",
-  "module": "./esm/index.mjs",
   "types": "./cjs/index.d.ts",
+  "main": "./dist/bundle.umd.cjs",
```

Comment:
First:
I'm pretty sure that would break cjs builds.
Second:
You don't have that file at all in the dist.